agent/csi/mod_csi_docker.go

```diff
@@ -386,25 +579,50 @@ func (d *dockerBridge) List() (*dvol.ListResponse, error) {
 			continue
 		}
 
-		vol := &dvol.Volume{Name: name}
+		if _, ok := volMap[name]; ok {
```

Comment:
Looks like this came back in. Sneaky code...  :)

Reply:
Did you remove it? If you did then I'm concerned that the patch didn't catch that. If everything else looks fine I'll chalk it up to my error.

Reply:
Wait...